### ISSUE TITLE
Don't need callback?

### ISSUE THREAD

**Issue by jianlong-yuan** [VainF/Pytorch-Synchronized-BatchNorm#1]
When we use your code, do we need call back?
```
		net = nn.DataParallel(net)
		patch_replication_callback(net)
```
